transcom/mymove
# B 17814 rename military bases

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/808731)

## Summary

See the agility ticket for a more in-depth explanation from Daniel, but TLDR: the `addresses` table was not updated along with the duty locations when there were some base name changes. This migration sets the correct `City` name in the addresses table for those that need updated. It also creates an entry for Fort Barfoot, previously Fort Pickett, that did not have an entry in the `duty_location_names` table.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the milmove application.
2. Login as a customer and start the process of creating a move.
3. Type in one of the duty locations that was previously displaying an incorrect value, such as Fort Cavazos. It should display Fort Cavazos underneath when selected. See the attachment on the Agility ticket titled 'Current Duty Stations displaying incorrectly' for more examples to test.

### Database

#### Any new migrations/schema changes:


## Screenshots
